addons/website/models/mixins.py

```diff
@@ -126,7 +126,7 @@ class WebsitePublishedMixin(models.AbstractModel):
     website_published = fields.Boolean('Visible on current website', related='is_published', readonly=False)
     is_published = fields.Boolean('Is Published', copy=False, default=lambda self: self._default_is_published())
     can_publish = fields.Boolean('Can Publish', compute='_compute_can_publish')
-    website_url = fields.Char('Website URL', compute='_compute_website_url', help='The full URL to access the document through the website.')
+    website_url = fields.Char('Website URL', compute='_compute_website_url', help='The full URL to access the document through the website.', translate=True)
```

Comment:
The field is not *translatable* IMHO.  But its value depends on the context lang.
So I recommend to add `@api.depends_context('lang')` on the compute method, instead.